extensions/amp-auto-lightbox/0.1/amp-auto-lightbox.js

```diff
@@ -414,7 +425,11 @@ export function scanDoc(ampdoc) {
 
 AMP.extension(TAG, '0.1', ({ampdoc}) => {
   ampdoc.whenReady().then(() => {
-    ampdoc.getRootNode().addEventListener(AmpEvents.DOM_UPDATE, () => {
+    ampdoc.getRootNode().addEventListener(AmpEvents.DOM_UPDATE, ({target}) => {
+      // Prevent recursion.
+      if (target.hasAttribute(LIGHTBOXABLE_ATTR)) {
+        return;
+      }
```

Comment:
I think it is enough to just scan the `descendants` of `target`.

Reply:
That's much better, changed.